### PR TITLE
chore: update tests to CFT 1.5 and project-factory ~> 13.0

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: test/fixtures/cloudbuild_enabled/versions.tf [hunk 1]
```diff
@@ -15,5 +15,5 @@
  */
 
 terraform {
-  required_version = ">=0.12.20"
```
Comment (bharathkkb) [inline test/fixtures/cloudbuild_enabled/versions.tf:18]: nit: unrelated but we can actually remove this from fixtures since its just for our usage.

**Review by bharathkkb — APPROVED** [review]

File: test/fixtures/cloudbuild_enabled/versions.tf (hunk 1 above)
Comment (apeabody) [reply test/fixtures/cloudbuild_enabled/versions.tf:18]: SG - I'll remove the entire file as this is the only item in them.

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@apeabody
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**